ui/src/duck/designation.ts

```diff
@@ -1,37 +1,31 @@
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 
-import { Designation, fetch } from 'src/entities/Designation';
+import { Designation, fetchMany } from 'src/entities/Designation';
 
 export interface DesignationState {
-  error?: string;
-  isLoading: boolean;
+  isFetching: boolean;
   designation: Designation[];
 }
 
 export const fetchDesignation = createAsyncThunk<
   Designation[],
 undefined,
 { rejectValue: Error; state: DesignationState }
-  >('designations/fetch', fetch);
+  >('designations/fetch/many', fetchMany);
```

Comment:
Using `/` is akin to adding a namespace. `fetch/many` doesn't look like a valid case for using a namespace. I recommend we make it `designations/fetchMany` (and `designations/fetchOne`)

Reply:
alright that's make sense